src/CommonGLPI.php

```diff
@@ -138,6 +138,26 @@ public static function getType()
         return get_called_class();
     }
 
+    public static function isFormActionAllowed(string $action): bool
+    {
+        return \in_array($action, static::getAllowedFormActions(), true);
+    }
+
+    /**
+     * Used by generic forms for details and logging purposes
+     *
+     * @return string
+     */
+    public static function getLogServiceName(): string
+    {
+        return '';
+    }
+
+    public static function getLogLevel(): int
+    {
+        return 4;
```

Comment:
4 is the default almost everywhere, BUT there are a few cases where it's another value from 1 to 5.

Reply:
There should probably be consts/enum for these levels.

Reply:
Yep, definitely! This PR is already big enough, so maybe another PR could introduce a special interface with the constants in it 👌